scrapers.js

```diff
@@ -1148,6 +1148,53 @@ let scrapers = [
       return counties
     }
   },
+  {
+    state: 'TN',
+    country: 'USA',
+    url: 'https://www.tn.gov/health/cedep/ncov.html',
+    scraper: async function() {
+      let counties = [];
+      let $ = await fetch.page(this.url);
+      let $table = $('th:contains("Case Count")').closest('table');
+
+      let $trs = $table.find('tbody > tr');
+
+      $trs.each((index, tr) => {
+        if (index < 1) {
+          return;
+        }
+        let $tr = $(tr);
+        counties.push({
+          county: parse.string($tr.find('td:first-child').text()) + ' County',
+          cases: parse.number($tr.find('td:last-child').text())
+        });
+      });
+      return counties;
+    }
+  },
+  {
+    state: 'OH',
+    country: 'USA',
+    url: 'https://odh.ohio.gov/wps/portal/gov/odh/know-our-programs/Novel-Coronavirus/welcome/',
+    scraper: async function() {
+      let counties = [];
+      let $ = await fetch.page(this.url);
+      let $paragraph = $('p:contains("Number of counties with cases:")').text();
+      let regExp = /\(([^)]+)\)/;
+      let parsed = regExp.exec($paragraph);
+      let arrayOfCounties = parsed[1].split(',');
+      arrayOfCounties.map(county => {
+        let splitCounty = county.trim().split(' ');
+        counties.push({
+          county: splitCounty[0] + ' County',
```

Comment:
throw a `parse.string(splitCounty[0])` in to clean out whitespace, whack random zero-width chars, etc.

Reply:
Added